src/chart/generator/guides.cpp

```diff
@@ -31,14 +31,14 @@ bool GuidesByAxis::operator==(const GuidesByAxis &other) const
 void Guides::init(const Axises &axises, const Options &options)
 {
 	auto isCircle =
-	    options.shapeType.getFactor(ShapeType::Type::circle);
-	auto isLine = options.shapeType.getFactor(ShapeType::Type::line);
-	auto isHorizontal = options.horizontal;
+	    options.shapeType.factor<Math::FuzzyBool>(ShapeType::circle) != false;
```

Comment:
options.shapeType.get() != ShapeType::circle

Reply:
In this function, nothing got interpolated yet, FuzzyBool::operator bool() and Interpolated::get() can be called safely.

Reply:
done